src/global/global_init.cc

```diff
@@ -571,15 +563,10 @@ void global_init_chdir(const CephContext *cct)
   }
 }
 
-/* Map stderr to /dev/null. This isn't really re-entrant; we rely on the old unix
```

Comment:
same here: I would prefer such comments to remain

Reply:
Since 3c2d91ed11c19ee8769acef8268938d2cab59152, we no longer use the trick of close-then-re-open, instead, we use `dup2`, which does not rely on the "old unix behavior that the file descriptor that gets assigned is the lowest available one." So these comment is out-of-date and inaccurate now.

I will update the commit message to include more details. And add back some of these that are still valid.